internal/controllers/import_controller_test.go

```diff
@@ -57,38 +59,47 @@ var _ = Describe("reconcile CAPI Cluster", func() {
 	)
 
 	BeforeEach(func() {
+		var err error
+
+		ns, err = testEnv.CreateNamespace(ctx, "commonns")
```

Comment:
I had to read the name of the namespace twice, i thought there was a typo :wink:

Reply:
It is just a generated name for the namespace, and naming is hard…